src/containers/ListingPage/ListingPage.jsx

```diff
@@ -51,6 +53,14 @@ const ListingPage = props => {
     }
   }, [queryParams.subjects]);
 
+  useEffect(() => {
+    if (md === null) {
+      const markdown = new Remarkable();
```

Comment:
Hvorfor vil vi at `md` skal ligge i state og oppdateres med `useEffect`?

Reply:
Var berre for å unngå å opprette ny Remarkable for kvar render.